tests/test_github_app_auth.py

```
@@ -0,0 +1,40 @@
+from ghdcbot.adapters.github.app_auth import (
+    DynamicBearerAuth,
+    build_github_httpx_client,
+    resolve_github_token,
+)
+
+
+def test_resolve_github_token_falls_back_to_pat(monkeypatch: pytest.MonkeyPatch) -> None:
+    monkeypatch.delenv("GITHUB_APP_ID", raising=False)
+    monkeypatch.delenv("GITHUB_APP_INSTALLATION_ID", raising=False)
+    token = resolve_github_token(pat="ghp_test_pat")
+    assert token == "ghp_test_pat"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "token"

(S105)